python_liftbridge/python_liftbridge.py

```diff
@@ -107,4 +107,12 @@ def _subscribe_request(self, stream):
             )
 
     def _create_publish_request(self, message):
-        return python_liftbridge.api_pb2.PublishRequest(message=message)
+        return python_liftbridge.api_pb2.PublishToSubjectRequest(
+            key=message.key, 
+            value=message.value,
+            subject=message.subject,
+            headers=message.headers,
+            ackInbox=message.ackInbox,
+            correlationId=message.correlationId,
+            ackPolicy=message.ackPolicy,
+        )
```

Comment:
`PublishToSubjectRequest` here seems to publish direct to NATS subject. I guess here we want to publish to Liftbridge's stream instead.

Something roughly like like

```
def _create_publish_request(self, message):
        publish_request_option = {
            "stream": message.stream,
            "value": message.value,
            "partition": message.partition
        }
        try:
            publish_request_option["key"] = message.key
        except AttributeError:
            pass
        try:
            publish_request_option["ackInbox"] = message.ack_inbox
        except AttributeError:
            pass
        try:
            publish_request_option["correlationId"] = message.correlation_id
        except AttributeError:
            pass
        try:
            publish_request_option["ackPolicy"] = message.ack_policy
        except AttributeError:
            pass
        return python_liftbridge.api_pb2.PublishRequest(**publish_request_option)

```

In the [API](https://github.com/liftbridge-io/liftbridge-api/blob/f32e77f70d8d507557885fe609eaf5fc28e548ee/api.proto#L76), it looks like this one.

Reply:
Again, the same concern here about that message class uses `subject` instead of the `name` to publish the message to the stream.